ImGuiApp/ImGuiController/ImGuiController.cs

```diff
@@ -427,6 +456,61 @@ private static ImGuiKey TranslateInputKeyToImGuiKey(Key key)
 		};
 	}
 
+	private static bool TryGetImGuiMouseButton(MouseButton button, out ImGuiMouseButton imguiMouseButton)
```

Comment:
Rename to `TranslateInputMouseButtonToImGuiMouseButton` to match `TranslateInputKeyToImGuiKey`

Reply:
Make this method use a switch expression and throw on unknown values.

It should handle all valid cases so the TryGet pattern is not needed.

Reply:
Silk.NET supports many more mouse buttons than ImGui itself (eg. MouseButton4 -> 10). 

I'm thinking an alternate implementation like you are suggesting would return valid values for Left/Right/Middle and Count for every other case. If Count is returned then the usage would do nothing.

Reply:
As there are only 12 mouse buttons supported it is easy to implement each case statement and add the throw on unknown.

Reply:
Implemented changes as I've described above.